es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchDAOV5.java

```diff
@@ -149,6 +150,19 @@ public ElasticSearchDAOV5(Client elasticSearchClient, ElasticSearchConfiguration
                     logger.warn("Request {} to async dao discarded in executor {}", runnable, executor);
                     Monitors.recordDiscardedIndexingCount();
                 });
+
+        corePoolSize = 1;
+        maximumPoolSize = 2;
+        keepAliveTime = 30L;
+        this.logExecutorService = new ThreadPoolExecutor(corePoolSize,
```

Comment:
Hi @apanicker-nflx We need same changes in ELasticSearchRestDAOV5 and ELasticSearchRestDAOV6